### DISCUSSION TITLE
Incorrect campus location

### DISCUSSION THREAD

**Discussion by mtaku3** [community/community#53260] [GitHub Education]
I got a email that says my application has been rejected for these reasons.

- The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

- You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

Although, I live close to the campus. I thought that the incorrect GPS location had been sent to GitHub. So I canceled my application and re-sent an application. At that time, I inspected network traffics and confirmed that the correct GPS location being sent.
**So I'm pretty sure that the campus's location has been registered wrongly in GitHub's database.**

How can I solve the issue? **Thanks for anyone's kind and help**

**Comment by samus-aran** [comment]
Hi @mtaku3 

If you believe that your applied school's details are incorrect please raise a ticket with [Education support](https://support.github.com/contact/education) using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Reply by mtaku3** [reply]
Hi,
I've already submitted a ticket about the issue from the form. But they immediately closed the ticket without providing any information. So I just ended up posting about the issue on this community discussion.
![IMG_0797](https://user-images.githubusercontent.com/57391256/234012337-f03ed6c4-3964-46a2-8708-356e99b3ae48.jpeg)

**Reply by mtaku3** [reply]
I made another ticket again and hope for them to get in touch with me for this time, not closing the ticket again 😅
I’ll let you know if they did that because I don’t have another option.

**Reply by mtaku3** [reply]
Finally I got the reply from the support and they told me that the campus location for *National Institute of Technology, Kagoshima College* is set for United States.
I don't know why, but they just reviewed my application and closed the ticket without mentioning that the campus location is wrongly set.
But I re-opened the ticket and asked to fix it. Hope this solves it..

**Reply by mtaku3** [reply]
It has been almost 2 months and finally, my application have been approved!
